config/xsl/common/util/load-profile-table.xsl

```diff
@@ -262,6 +264,11 @@
                                     </xsl:if>
                                     <xsl:value-of select="numberOfUsers"/>
                                 </td>
+                                <td class="value number">
+                                    <xsl:text>(</xsl:text>
+                                    <xsl:value-of select="format-number(numberOfUsersPercentage, '#,##0.00')" />
+                                    <xsl:text>%)</xsl:text>
```

Comment:
Don't enclose the percentage value in parentheses if we display it in a separate column.